.github/workflows/tests.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: ["3.8"]
-        django: ["22","32"]
+        python-version: ["3.8", "3.10", "3.11"]
```

Comment:
Which criteria you're using here in order to add 3.10 and 3.11? should we adopt it in the other plugins?

Reply:
@mariajgrimaldi no, we spoke with Felipe about this and the "support windows file" and he explained me how to read the file and 3.10 and 3.11 is not necessary, so we can remove them, however I think that is a good practice fix the deprecations warnings to doesn't have problems in a future.